x-pack/plugins/maps/common/elasticsearch_util/elasticsearch_geo_utils.test.js

```diff
@@ -397,7 +397,7 @@ describe('createExtentFilter', () => {
         minLat: 35,
         minLon: -89,
       };
-      const filter = createExtentFilter(mapExtent, geoFieldName, 'geo_point');
+      const filter = createExtentFilter(mapExtent, geoFieldName);
```

Comment:
Can you remove the 'geo_point field' and 'geo_shape field' describe blocks under 'createExtentFilter' block. It does not make sense to split the tests into blocks for geo_point and geo_shape now that geoFieldType is no longer passed into the method

Reply:
Can you rename the test description "should return elasticsearch geo_bounding_box filter for geo_point field" to just "should return elasticsearch geo_bounding_box filter"